### PR TITLE
Update manage.sh for updated aws cli for issue 10480

### PR DESCRIPTION
**Pull request by joewhite101**
This is a recreation of an old pull request that I broke with a bad merge.

Old PR: https://github.com/quarkusio/quarkus/pull/10481

This resolves an issue where the aws cli expects the input format of the json to be base64. This change allows the input to be raw json so that the manage script works.The current script results in:

/projects/quarkus/lamda-test$ target/manage.sh invoke
Invoking function
++ aws lambda invoke response.txt --function-name LamdaTest --payload file://payload.json --log-type Tail --query LogResult --output text
++ base64 -D

An error occurred (InvalidRequestContentException) when calling the Invoke operation: Could not parse request body into json: Unexpected character ((CTRL-CHAR, code 157)): expected a valid value (number, String, array, object, 'true', 'false' or 'null')
 at [Source: (byte[])"���&����y"; line: 1, column: 2]

### CODE REVIEW

**Comment by joewhite101** [comment]
@oztimpower I updated the script to work with both 1.x and 2.x but had to recreate the PR. I somehow messed up my fork and pushed a bunch of stuff we didn't want into that PR so I closed that one and created a new one.

**Review by gsmet — DISMISSED** [review]
This will require a proper squash but I will do it once it's approved if you struggle with git :).

**Comment by joewhite101** [comment]
@gsmet Thanks, that may be good. I tried to rebase and made it worse. Need to clean up my fork. If you prefer that i close this and start over let me know and i'll create a clean one. Shows clean in local but github disagrees.

**Comment by gsmet** [comment]
I cleaned up the PR. @oztimpower or @patriot1burke could you have a look?

**Comment by joewhite101** [comment]
We should probably link issue as well:

https://github.com/quarkusio/quarkus/issues/10480

**Review by patriot1burke — APPROVED** [review]

**Comment by patriot1burke** [comment]
Thanks Joe!

**Pull request merged**
